### ISSUE TITLE
function 'anonymous lambda' called with unexpected argument 'vaapiIntel'

### ISSUE THREAD

**Issue by brianmay** [guibou/nixGL#136]
On Rasberry Pi 3:

```
$ nix run github:guibou/nixGL#nixGLIntel
error:
       … while evaluating a branch condition

         at /nix/store/kc7nh9sdr4z8c7xc9j9891v212k2zj8k-source/lib/customisation.nix:81:7:

           80|     in
           81|       if builtins.isAttrs result then
             |       ^
           82|         result // {

       … while calling the 'isAttrs' builtin

         at /nix/store/kc7nh9sdr4z8c7xc9j9891v212k2zj8k-source/lib/customisation.nix:81:10:

           80|     in
           81|       if builtins.isAttrs result then
             |          ^
           82|         result // {

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: function 'anonymous lambda' called with unexpected argument 'vaapiIntel'

       at /nix/store/gb1rfp8s0mhvjwhv33amfjd3cpzrigj0-source/nixGL.nix:1:1:

            1| { # # Nvidia informations.
             | ^
            2| # Version of the system kernel module. Let it to null to enable auto-detection.
```

**Comment by brianmay** [comment]
I think this was broken in https://github.com/guibou/nixGL/commit/6ceeca1bcded73863af32f2dff90cd3a2a24a9ea

At least going back to the previous commit works,

**Comment by SuperSandro2000** [comment]
should be fixed with #137
